tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es6",
     "module": "commonjs",
     "declaration" : true,
-    "sourceMap": false,
+    "sourceMap": true,
```

Comment:
Why this change?

Reply:
was having an issue producing jest inline snapshots https://github.com/facebook/jest/issues/6744#issuecomment-861722016